migration/sql-files/061-replace-index-space-name.sql

```diff
@@ -1,4 +1,14 @@
 -- drop existing unique index
 DROP INDEX spaces_name_idx;
+-- rename duplicate spaces in existence and keep only one as it was
+UPDATE spaces SET name = CONCAT(lower(name), '-renamed')
```

Comment:
What if there's already something called "*-renamed" :) Practically I doubt it would be present, but technically that could cause us trouble, and a unique ID would be safe.